tests/test_translator.py

```diff
@@ -108,6 +112,7 @@ def test_path_to_local_track_uri(path, uri):
         (b"foo", "local:directory:foo"),
         ("æøå", "local:directory:%C3%A6%C3%B8%C3%A5"),
         (b"\x00\x01\x02", "local:directory:%00%01%02"),
```

Comment:
I got rid of all these bytes `path` values (in all tests) and made everything a `Path` since that's the only thing we call these with.

Reply:
Thanks! Applied in the last commit now.